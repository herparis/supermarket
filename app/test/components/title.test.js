import React from 'react'
import { shallow } from 'enzyme'

import Title from '../../src/components/title/title'

function checkRender(wrapper) {
  expect(wrapper.isEmptyRender()).toBe(false)
  expect(wrapper).toMatchSnapshot()
}

describe('Title components rendeding', () => {
  const title = shallow(<Title />)
  it('title renders', () => {
    checkRender(title)
  })
})