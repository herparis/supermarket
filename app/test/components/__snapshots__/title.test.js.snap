// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title components rendeding title renders 1`] = `ShallowWrapper {}`;
